### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OmarMalik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # credentials-io
 Library for reading/writing credentials
+
+## API
+
+### verify
```
Comment (msporny) [inline README.md]: We'll want to also specify the arguments to each of these functions and the expected return type or what is sent to the callback.

Comment (msporny) [inline README.md]: Check here for a more complete example: https://github.com/digitalbazaar/credentials-polyfill/blob/master/README.md#documentation

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+ credentials: {}
+}
+```
+
+### send(options, callback)
+Sends credentials to a URL using http-signatures for authentication.
+Options parameter should include following properties:
+```
+{
+  url,
```
Comment (msporny) [inline README.md]: What should the values be for these properties?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
 # credentials-io
 Library for reading/writing credentials
+
+## API
+
+### verify(identity, options, callback)
```
Comment (msporny) [inline README.md:6]: What does an identity object look like? What are the options you can pass the function? What's the function signature for the callback - callback(err, identity)?

Look at this for an example on how to document parameters: https://github.com/digitalbazaar/credentials-polyfill/blob/master/README.md#registering-a-decentralized-identifier

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+ identity: <framed identity>,
+ domain: <domain the signature was signed for>,
+ publicKey: <public key object for the creator of the signature>,
+ publicKeyOwner: <identity object for the creator of the signature>,
+ // a map of verified objects for each credential (ID => verified)
+ credentials: {}
+}
+```
+
+### send(options, callback)
```
Comment (msporny) [inline README.md:28]: These parameters need to be documented. Look at this for an example on how to document parameters: https://github.com/digitalbazaar/credentials-polyfill/blob/master/README.md#registering-a-decentralized-identifier

File: README.md [hunk 5]
```diff
@@ -1,2 +1,37 @@
...
+```
+{
+  url,
+  credential,
+  privateKeyPem,
+  publickKeyId
+}
+```
+
+### checkDomain(domain, options, callback)
```
Comment (msporny) [inline README.md]: If this call isn't a part of the public API, which I don't think it is, then there is no need to document it (although, we may want to rename it to _checkDomain). You'll want to talk to @dlongley about this.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,37 @@
...
+(not implemented) Checks against some config value for the domain
+
+### checkKeyOwner(owner, key, options, callback)
```
Comment (msporny) [inline README.md]: If this call isn't a part of the public API, which I don't think it is, then there is no need to document it (although, we may want to rename it to _checkDomain). You'll want to talk to @dlongley about this.

File: README.md (hunk 4 above)
Comment (dlongley) [inline README.md:28]: Line 8 has a typo that could be fixed at the same time (recieves => receives).

**Pull request merged**